packages/components/src/Link/Link.test.tsx

```diff
@@ -0,0 +1,43 @@
+import { useEffect, useRef } from 'react';
+import { renderWithTheme, screen } from '@astral/tests';
+
+import { Link } from '.';
+
+describe('Link', () => {
+  it('Ref доступен', () => {
+    const resultRef = { current: null };
+
+    const LinkWithRef = () => {
+      const ref = useRef(null);
+
+      useEffect(() => {
+        resultRef.current = ref.current;
+      }, []);
+
+      return <Link ref={ref} />;
+    };
+
+    renderWithTheme(<LinkWithRef />);
+    expect(resultRef?.current).not.toBeNull();
+  });
+
+  it('Иконка слева отображается', () => {
+    const label = 'Link';
+
+    renderWithTheme(<Link children={label} withAdornment="start" />);
```

Comment:
```suggestion
    renderWithTheme(<Link withAdornment="start">Link</Link>);
```

+ не указал href, смотри ошибки в ci